src/main/java/mate/academy/model/MovieSession.java

```diff
@@ -0,0 +1,66 @@
+package mate.academy.model;
+
+import java.time.LocalDateTime;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.ManyToOne;
+
+@Entity
```

Comment:
lets add `@Table` to specify the table name used for this entity, same for the other one